### ISSUE TITLE
SENet incorporation with other nets

### ISSUE THREAD

**Issue by leochli** [farmingyard/SENet#3]
Hi @farmingyard ,

Have you tried combining SENet module with other popular structures (except Resnext), for instance, MobileNet, ShuffleNet etc.

thx!

**Comment by farmingyard** [comment]
@leolee96 
Yes, I have combined it with mobilenet, and the result is very impressive, you can try it. If it is helpful, you can give a star, ^.^

**Comment by leochli** [comment]
Hi @farmingyard ,

Thx for your sharing, nice job! I'm training it with mobilenet now. Btw, the training speed is 2 times slower in my case. It's probably caused by the 2 extra fc layer.

Do you have any ideas on how to train faster?

thx!

**Comment by hujie-frank** [comment]
Hi @leolee96 
        The overheads most result from the global average pooling implemented by cuDNN. Maybe you can follow this repository with some optimization.
   ` https://github.com/hujie-frank/SENet`

**Comment by leochli** [comment]
Hi @hujie-frank ,
Great job!  Thanks!

**Comment by farmingyard** [comment]
@hujie-frank 
Hi jie，nice job！The se module is very effective，thanks！

**Comment by jinjunbo** [comment]
@farmingyard hello , can you share the pretrain model and deploy.txt with mobilenet+SE? Thanks !

**Comment by MingChaoXu** [comment]
@jinjunbo @farmingyard I am also interested in the deploy.txt with mobilenet+SE ? Have you worked it out ? Thanks !

**Comment by jinjunbo** [comment]
@MingChaoXu Yes,have a good results with mobilenet+SE on remote sensing image target detection,you can try it out.
